packages/graphql/src/mutations/identity/deployIdentity.js

```diff
@@ -9,10 +9,14 @@ import costs from '../_gasCost.js'
 
 async function deployIdentity(
   _,
-  { from = contracts.defaultMobileAccount, profile = {}, attestations = [] }
```

Comment:
why this change?

Reply:
Default values are returned only if `undefined` is passed ([Ref.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters)). For the `identity.attestations` and `identity.profile`, we get `null` from the GraphQL resolver.